bundles/org.openhab.binding.tado/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -197,18 +193,21 @@
 		<item-type>Switch</item-type>
 		<label>Swing</label>
 		<description>State of AC swing (only if supported by AC)</description>
+		<category>flow</category>
 	</channel-type>
 
 	<channel-type id="light">
 		<item-type>Switch</item-type>
 		<label>Light</label>
 		<description>State of control panel light (only if supported by AC)</description>
+		<category>light</category>
```

Comment:
The practice is to have an uppercase letter as first letter for the category.
If you can change that to be homogenous with all the other bindings.

Reply:
Ok. Done.